hack/e2e-kueueviz-frontend.sh

```diff
@@ -37,7 +37,8 @@ npm install
 npm start & FRONTEND_PID=$!
 
 # Run Cypress tests for kueueviz frontend
+cd ../../../test/e2e/kueueviz
```

Comment:
Actually, can we use PROJECT_DIR for this purpose? Similarly above `cd cmd/kueueviz/frontend` we could probably resolve using PROJECT_DIR